src/Android/MainActivity.cs

```diff
@@ -133,6 +138,21 @@ protected override void OnResume()
                 .GetResult();
         }
 
+        // TODO: When #1660 gets merged then we can use the task extension FireAndForget instead of this method
```

Comment:
Another option (not necessarily better, just a different approach) for this is to expose a synchronous method that saves the task internally, tend to prefer this approach like this to hidden tasks, easier to debug and figure out when things aren't working. 

`ThemeManager.UpdateThemeOnPages();`

```
public void UpdateThemeOnPages()
{
    _updateThemeTask = UpdateThemeOnPagesAsync();
}
```